doc/rtd/topics/datasources/exoscale.rst

```diff
@@ -38,8 +38,9 @@ Configuration
 Users of this datasource are discouraged from changing the default settings
 unless instructed to by Exoscale support.
 
-The following settings are available and can be set for the datasource in
-system configuration (in `/etc/cloud/cloud.cfg.d/`).
+The following settings are available and can be set for the
+:ref:`datasouce system configuration<datasource_system_config>`
```

Comment:
Since we are agreed on Base Configuration instead of System config, let's keep our verbiage (and link anchors) to "datasource base config" instead of "datasource system config". Also I had a typo "datasouce" instead of "datasource".
Pushed this tiny commit on top of yours to sort typo and cross link.